vent/main.py

```diff
@@ -19,7 +21,22 @@ def parse_cmd_args():
     return parser.parse_args()
 
 
+def init_logger():
```

Comment:
This does, aside from removing gui handlers, the same as init_logger() in rpc.py, doesn't it? Maybe define once, and reuse the same function?